test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,32 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Bind an imported Node.js application to Etcd database
+        Given Etcd operator running
+        Given Etcd cluster "etcd-cluster-example" is created
```

Comment:
rename to `Etcd cluster "etcd-cluster-example" is running`

Reply:
done